src/Codeception/Module/REST.php

```diff
@@ -498,7 +498,13 @@ protected function execute($method, $url, $parameters = [], $files = [])
             $this->debugSection("Request", "$method $url " . $requestData);
             $this->response = (string)$this->connectionModule->_request($method, $url, [], $files, [], $parameters);
         }
-        $this->debugSection("Response", $this->response);
+        $response = $this->response;
```

Comment:
Yeah that makes sense.

I'll also rework it to move the formatting out to a couple of methods, so that the code is not repeated.